cluster/charts/crossplane/templates/rbac-manager-clusterrole.yaml

```diff
@@ -0,0 +1,60 @@
+{{- if .Values.rbacManager.deploy }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "name" . }}-rbac-manager
+  labels:
+    app: {{ template "name" . }}
+    chart: {{ template "chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - events
+  verbs:
+  - create
+  - update
+  - patch
+  - delete
+- apiGroups:
```

Comment:
Do we still need to have RBAC on namespaces when management policy is `Basic`?

Reply:
No, and nor would we need to do anything with `roles`. Are you thinking we should put this behind a conditional so that the RBAC manager gets less access when it's running in basic mode?

Reply:
Exactly 👍

Reply:
I'm inclined to avoid doing this, unless you feel strongly. It feels like the templating logic could get hairy fast as we introduced new RBAC management policies. I think with the current implementation we'd have to wrap these rules in conditionals like `if managementPolicy != Basic` or `if managementPolicy == "" || managementPolicy == All`.

That all said, if running with these lower privileges did turn out to be important to the community I'd be happy to make the changes.

Reply:
I am fine deferring this for now :+1: